lib/roadie/rails.rb

```diff
@@ -3,6 +3,8 @@ module Rails
   end
 end
 
+require "active_support/concern"
```

Comment:
It's just that standard Ruby handles this pretty easily itself without the extra sugar (which feels like adding sugar to your soda IMO, maybe it's a bit sweeter but I don't see the point right here).

---

**Strawman time**

It's a bit like having an ActiveSupport library that overrides the `new` method on classes like this:

```class
module NewWithOptionalBlock
  def new(*)
    instance = super
    if block_given?
      yield instance
    end
    instance
  end
end
```

I just don't see the point in using an external library for this pattern when just adding it straight to your class would do. Just accept a block in `initialize` and be done with it.

---

But again, I cannot say a good reason *not* to do it here as we're in Rails, and the argument that it's so easy to implement *without* Rails means that if Rails ever removes it, it's very easy to remove again. It's a double-edged sword that way. :-)